docs/monitoring.md

```diff
@@ -222,17 +222,33 @@ both running applications, and in the history server.  The endpoints are mounted
 for the history server, they would typically be accessible at `http://<server-url>:18080/api/v1`, and
 for a running application, at `http://localhost:4040/api/v1`.
 
+In the API, an application is referenced by its application ID, `[app-id]`.
+When running on YARN, each application may have multiple attempts; each identified by their `[attempt-id]`.
+In the API listed below, `[app-id]` will actually be `[base-app-id]/[attempt-id]`,
+where `[base-app-id]` is the YARN application ID.
+
 <table class="table">
   <tr><th>Endpoint</th><th>Meaning</th></tr>
   <tr>
-    <td><code>/applications</code></td>
-    <td>A list of all applications</td>
+    <td ><code>/applications</code></td>
+    <td>A list of all applications
+    <br>
+    <code>?status=[completed|running]</code> list only applications in the chosen state
+    <br>
+    <code>?minDate=[date]</code> earliest date/time to list. Examples:
+    <br><code>?minDate=2015-02-10</code>
+    <br><code>?minDate=2015-02-03T16:42:40.000GMT</code>
+    <br><code>?maxDate=[date]</code> latest date/time to list; uses same format as <code>minDate</code></td>
   </tr>
   <tr>
     <td><code>/applications/[app-id]/jobs</code></td>
     <td>A list of all jobs for a given application</td>
   </tr>
   <tr>
+    <td></td>
+    <td><code>?status=[complete|succeeded|failed]</code> list only jobs in the state</td>
```

Comment:
I do think the form with one `<td>` looks meaningfully better. Otherwise these looks like hanging docs for some unspecified endpoint